templates/ntp.conf.epp

```diff
@@ -0,0 +1,113 @@
+# ntp.conf: Managed by puppet.
+#
+# Enable next tinker options:
+# panic - keep ntpd from panicking in the event of a large clock skew
+# when a VM guest is suspended and resumed;
+# stepout - allow ntpd change offset faster
+<% if $ntp::_tinker and ($ntp::_panic or $ntp::stepout) {-%>
+tinker<% if $ntp::_panic { %> panic <%= $ntp::_panic %><% } %><%if $ntp::stepout { %> stepout <%=$ntp::stepout %><% } %>
+<% } -%>
+
+<% if $ntp::disable_monitor {-%>
```

Comment:
I haven't seen this construct in use before and `{-%>` is an aummm... "unusual" sigil sequence. I understand how it got there but, wow.  // cc @loriland